pkg/registry/experimental/controller/etcd/etcd.go

```diff
@@ -88,6 +88,7 @@ func (r *ScaleREST) Update(ctx api.Context, obj runtime.Object) (runtime.Object,
 		return nil, false, errors.NewNotFound(extensions.Resource("replicationcontrollers/scale"), scale.Name)
 	}
 	rc.Spec.Replicas = scale.Spec.Replicas
+	rc.ResourceVersion = scale.ResourceVersion
```

Comment:
@liggitt If scale.ResourceVersion is empty string, this should result in an unconditional update, whereas if we didn't copy the empty string, the user could get spurious update failures.